prove/Develop05/GoalTracker.cs

```diff
@@ -33,13 +34,21 @@ public void AddGoal(Goal goal)
 
     public void DisplayGoals()
     {
-        Console.WriteLine("The goals are:");
+        if (_goals.Count != 0)
+        {
+            Console.WriteLine("The goals are:");
 
-        for (int i = 0; i < _goals.Count; i++)
+            for (int i = 0; i < _goals.Count; i++)
```

Comment:
I like the foreach for these types of structures when you are iterating through a list of things.

Reply:
Fixed